test/README.md

```diff
@@ -0,0 +1,6 @@
+
+Spin up services: 
```

Comment:
Add documentation on how to run the unit and integration tests.

Reply:
Done